tests/Fixer/Phpdoc/PhpdocTypesOrderFixerTest.php

```diff
@@ -138,9 +138,19 @@ public static function provideFixWithNullFirstCases(): iterable
                 '<?php /** @var Foo[]|null|Foo|Foo\Bar|Foo_Bar */',
             ],
             [
-                '<?php /** @method void bar(null|string $delimiter = \',<br/>\') */',
```

Comment:
Isn't it a breaking change? If I see it correctly, the way how `@method void bar(string|null $delimiter = \',<br/>\')` is handled has changed and `string|null` won't be sorted anymore?

Reply:
see https://psalm.dev/docs/annotating_code/type_syntax/callable_types/ grammar (or phpstan grammar)

`$arg=` (or `$arg =`) represents optional parameter, `$arg = const` is not valid phpdoc

Reply:
`'@method with invalid 2nd phpdoc'` testcase tests the original phpdoc will be kept as-is correctly